### PR TITLE
Doc: Add information to README  that Heroku servers are unavailable

### LINKED ISSUE #113: Doc: Add information on README that Heroku servers are currently unavailable
## Description
As a maintainer,
I need to inform contributor that Heroku server is currently not available ,
so that I can make sure all contributors only use local server for development.

## Mocks
NA

### Update [Required]
- [ ] Add information that Heroku servers are currently unavailable and use Maya's [medium blog] as reference.
 
### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description
Add information that Heroku serverrs are currently unavailable due to [free dynos limitation](https://medium.com/anitab-org-open-source/why-you-cant-use-heroku-free-dynos-in-multiple-servers-dependent-application-fa1cdd9e9e07?source=friends_link&sk=281bb76050f052c03c6772e9be275f67)

Fixes #113 

### Type of Change:
- Documentation

**Code/Quality Assurance Only**
NA

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +42,11 @@ Follow the rest of the setup instructions (providing the environment credentials
...
+
+2. Even if we found the way to make Heroku servers work, please be aware that there are **drawbacks** if you chose to run the BIT and MS backend servers remotely (**option 2**):
 * There will be a **significant delay** from the time when the request is made on the BIT web GUI to the time the response is received. This is because BIT Heroku makes calls to MS-for-BIT Heroku backend server for MS related functionalities. Both of these Heroku servers are running on [free `dynos`](https://www.heroku.com/dynos) that have limited capacity. When you run both BIT and MS-for-BIT servers locally, you will not have this time lag issue.
-* The BIT Heroku baackend server will only carries the latest code that have been approved and merged to the develop branch. This means, if you are working on a PR or testing an open PR, this **Option 2** (running remote BIT Heroku server) **MUST NOT BE USED**.     
+* The BIT Heroku backend server will only carries the latest code that have been approved and merged to the develop branch. This means, if you are working on a PR or testing an open PR, this **Option 2** (running remote BIT Heroku server) **MUST NOT BE USED**.
```
Comment (rpattath) [inline README.md]: Please fix the typo "will only carries"

**Review by rpattath — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mtreacy002) [reply README.md]: Done 😉.

**Review by isabelcosta — APPROVED** [review]
Great update!

**Review by gaurivn — APPROVED** [review]
Good, nicely written

**Pull request merged**